src/idex/common.js

```diff
@@ -108,25 +108,27 @@ export function makeIdentityResolver (config, calls, cache) {
     }
 
     const responseReceived = (additionalParams, successCallback) => {
-      return response => {
+      return (responseText, response) => {
         let responseObj = {}
-        if (response) {
+        if (responseText) {
           try {
-            responseObj = JSON.parse(response)
+            responseObj = JSON.parse(responseText)
           } catch (ex) {
             console.error('Error parsing response', ex)
             fromError('IdentityResolverParser', ex)
           }
         }
-        cache.set(additionalParams, responseObj)
+
+        const expiresAt = responseExpires(response)
+
+        cache.set(additionalParams, responseObj, expiresAt)
```

Comment:
there will be no issue if the expiresAt is undefined?

Reply:
the cache handles it properly
`expiresAt || expiresInHours(defaultExpirationHours)`